recipes/libcurl/all/conanfile.py

```diff
@@ -313,30 +306,48 @@ def _patch_cmake(self):
         if not self._is_using_cmake_build:
             return
         cmakelists = os.path.join(self.source_folder, "CMakeLists.txt")
-        # Custom findZstd.cmake file relies on pkg-config file, make sure that it's consumed on all platforms
-        if self._has_zstd_option:
-            replace_in_file(self, os.path.join(self.source_folder, "CMake", "FindZstd.cmake"),
-                                  "if(UNIX)", "if(TRUE)")
         # TODO: check this patch, it's suspicious
         replace_in_file(self, cmakelists,
                               "include(CurlSymbolHiding)", "")
 
+        # brotli
+        replace_in_file(self, cmakelists, "find_package(Brotli QUIET)", "find_package(brotli REQUIRED CONFIG)")
```

Comment:
Why not use the Conan 1.55 feature and override this before the generate?

Reply:
It wouldn't work except for libnghttp2 and wolfssl.

Reply:
I wouldn't even do that for libnghttp2 because we want to link only nghttp2 component, so a patch is required anyway.

Reply:
🤮  that's sucky can you share your changes so I can flag it too the team please?

Reply:
Sorry I don't understand what you want me to share exactly. For me the reason why it can't work in this specific case is obvious when you know how `CMakeDeps` works and looking at what is replaced here in libcurl CMakeLists. It's a combination of casing issues and non-standard variables names.